src/main/docs/guide/httpServer/routing.adoc

```diff
@@ -117,6 +117,19 @@ The previous example uses the link:{api}/io/micronaut/http/annotation/Get.html[@
 
 NOTE: All the method annotations default to `/`.
 
+== @Options
+
+<<cors, CORS>> support handles OPTIONS preflight requests. However, if you want to dispatch OPTIONS requests without an Origin HTTP Header, you can enable it via:
+
+[configuration]
+----
+micronaut:
+  server:
+    dispatch-options-requests: true
```

Comment:
this should be under `CorsConfigration`

Reply:
That's doesn't need to related to CORS, Spring MVC has it the sam way

Reply:
This is not related to CORS or necessary for CORS.